### PR TITLE
Add blocking twitter; Fix #27

### LINKED ISSUE #27: Блочить твиттер

### PR DESCRIPTION
**Pull request by fyvfyv**

### CODE REVIEW

File: asocial/content/tw.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    float: none;
+    width: 100%;
+}
+
+@media(min-width: 1236px) {
+    body.three-col.win10-wrapper .asocial_block {
+        float: left;
+        margin: 0;
+    }
+}
```
Comment (isqua) [inline asocial/content/tw.css]: .

**Comment by isqua** [comment]
`¯\_(ツ)_/¯` 🎱

File: asocial/content/tw.css (hunk 1 above)
Comment (fyvfyv) [reply asocial/content/tw.css]: мм?

Comment (isqua) [reply asocial/content/tw.css]: there is no empty line in the end of file

**Pull request merged**